static/templates/homepage.html

```diff
@@ -0,0 +1,225 @@
+                            <div layout="col">Build your Career </div>
+                        </div>
+                        <br>
+                        <br>
+                        <div layout="row">
+                            Create your account now:
+                        </div>
+                        <br>
+                        <div layout="row">
+                            <md-button md-no-ink="" class="stylebuttonblue" ng-href="">FOR REQUESTERS</md-button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!--Ends Why Daemo Section-->
+
+
+        <!--Begins About Us Section-->
+        <div class="yellowbackground abouttext">
+            <div layout="row" layout-align="center center">
+                 <h1 align="center" class="text_title_hp">ABOUT US</h1>
+            </div>
+            <div layout="row" layout-align="center center" >
+                <div layout="col" layout-align=" center" class="text_hp margin_texta">
+                    Daemo was created via a world-wide, crowdsourced research process initi-
+                    <br>ated at Stanford University. We are a multidisciplinary group of researchers
+                    <br>who gathered together with the only purpose of creating Daemo as crowd-
+                    <br>sourcing platform that instills trust and fairness among its members.
+                </div>
+                    <div layout="col" layout-align="end center">
+                        <md-button md-no-ink="" class="stylebuttonblue" ng-href="">OUR PROFILES</md-button>
+                    </div>
+                
+            </div>
+        </div>
+        <div class="footerhp" >
+            <div layout="row" layout-align="center center">
+                 <h1 align="center" class="text_title_hp">ABOUT</h1>
+            </div>
+            <div layout="row" layout-align="center">
+                <div layout="col" layout-align="center start" class="footeralign">
+                    HOW IT WORKS
+                    <br>Profiles
+                    <br>Press
+                    <br>Terms of service
+                    <br>Privacy policy
+                </div>
+                <div layout="col" layout-align="center start" class="footeralign">
+                    CREATE ACCOUNT
+                    <br>For requesters
+                    <br>For workers
+                </div>
+                <div layout="col" layout-align="center start" class="footeralign">
+                    CONTACT
+                    <br>Contact
+                    <br>Support
+                    <br>Careers
+                </div>
+                <div layout="col" layout-align="center start" class="footeralign">
+                    GET STARTED
+                    <br>Find a job
+                    <br>Hire workers
+                </div>
+            </div>
+            <br>
+        </div>
+    </md-content>
+</div>
```

Comment:
having one class per element is not really good, try to re-use properties as much as possible